test/parallel/test-module-create-require-from-directory.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const path = require('path');
+
+const { createRequireFromPath } = require('module');
+
+const p = path.join(path.resolve(__dirname, '..', 'fixtures'), path.sep);
```

Comment:
Can you add another test without `path.sep` at the end?

Reply:
I've added a negative test 👍